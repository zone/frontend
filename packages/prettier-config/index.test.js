const prettier = require('prettier');
const { name: NPM_PACKAGE } = require('./package.json');
const config = require('./index.json');

describe(NPM_PACKAGE, () => {
  it('provides a config', () => {
    expect(config).toMatchSnapshot();
  });

  it('config is valid', () => {
    expect(() =>
      prettier.format('', { ...config, parser: 'babel' })
    ).not.toThrow();
  });
});
