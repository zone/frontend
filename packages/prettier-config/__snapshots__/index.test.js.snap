// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@zonedigital/prettier-config provides a config 1`] = `
Object {
  "arrowParens": "avoid",
  "singleQuote": true,
  "tabWidth": 2,
}
`;
